strongbox-parent/pom.xml

```diff
@@ -839,6 +839,13 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            
+            <dependency>
+                <groupId>org.javatuples</groupId>
+                <artifactId>javatuples</artifactId>
```

Comment:
Just for my personal understanding: Where is this being used?

Reply:
it's not used. Will be removed

Reply:
and now it's used :)
[here](https://github.com/strongbox/strongbox/pull/373/files#diff-166187c6b100d8cb032cf0c4edc81517L46)